frontendservice/src/views/CustomerHome.vue

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2020 Google LLC Licensed under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance with the License. You
+may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div class="home">
+    <h1>Welcome customer!</h1>
+    <p>
+      This page will be Home page for the customers. There will be a navbar
+      having the required navigation links myTokens etc and allShops will be
+      displayed with Search feature. Bavanya will be implementing this part.
```

Comment:
Let's try not to check-in names of expected contributors here.
If you want, you can add TODOs as comments where you can mention feature/bugs pending.

Reply:
Understood. Done.